### PR TITLE
[review-mirror] microsoft/PowerToys#49105

### PR DESCRIPTION
**Pull request by yeelam-gordon**
Review sandbox mirror of microsoft/PowerToys#49105 (branch `crutkas/peek-common-tests`). Opened by PR-autopilot for Copilot review + local build validation. Not for merge.

### CODE REVIEW

File: src/modules/peek/Peek.Common.UnitTests/MathHelperTests.cs [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+        public void Modulo_NegativeDividend_MinusTwo_ModThree_ShouldReturnOne()
+        {
+            Assert.AreEqual(1, MathHelper.Modulo(-2, 3));
+        }
+
+        /// <summary>
+        /// Product code: MathHelper.Modulo(int, int) — zero divisor
+        /// What: Documents that dividing by zero throws DivideByZeroException
+        /// Why: Edge case — callers must handle zero divisor; the method does not guard against it
+        /// </summary>
```
[the remark this answers is not in the document]
Comment (yeelam-gordon) [reply src/modules/peek/Peek.Common.UnitTests/MathHelperTests.cs:140]: Fixed in 3b69cc0bbf83b66c5191576ee386b222413387bb: updated the zero-divisor test XML doc to document the current `ArgumentOutOfRangeException` contract and positive-divisor guard. Validated with targeted Debug x64 build and vstest (37 passed).

File: src/modules/peek/Peek.Common.UnitTests/PathHelperTests.cs [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+        /// <summary>
+        /// Product code: PathHelper.IsUncPath(string)
+        /// What: Verifies null input returns false without throwing
+        /// Why: Null is a common edge case — Uri.TryCreate handles null gracefully
+        /// </summary>
+        [TestMethod]
+        public void IsUncPath_NullInput_ShouldReturnFalse()
+        {
+            Assert.IsFalse(PathHelper.IsUncPath(null));
```
[the remark this answers is not in the document]
Comment (yeelam-gordon) [reply src/modules/peek/Peek.Common.UnitTests/PathHelperTests.cs]: Fixed in 7462424a33dbefa43b53883fc8bf4ddcb52c4567: changed the intentional null test input to `null!`, matching the repo's nullable-test convention while keeping the product API unchanged. Validated with targeted Debug x64 build and vstest (37 passed).

File: src/modules/peek/Peek.Common.UnitTests/MathHelperTests.cs [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+        {
+            MathHelper.Modulo(5, 0);
+        }
+
+        [TestMethod]
+        [ExpectedException(typeof(ArgumentOutOfRangeException))]
+        public void Modulo_NegativeDivisor_ShouldThrow()
+        {
+            MathHelper.Modulo(-1, -3);
+        }
```
[the remark this answers is not in the document]
Comment (yeelam-gordon) [reply src/modules/peek/Peek.Common.UnitTests/MathHelperTests.cs:158]: Fixed in 7462424a33dbefa43b53883fc8bf4ddcb52c4567: added the missing XML doc block for the negative-divisor `ArgumentOutOfRangeException` test, consistent with the surrounding tests. Validated with targeted Debug x64 build and vstest (37 passed).

File: src/modules/peek/Peek.Common.UnitTests/MathHelperTests.cs [hunk 4]
```diff
@@ -0,0 +1,288 @@
...
+        public void Modulo_NegativeDividend_ExactMultiple_ShouldReturnZero()
+        {
+            Assert.AreEqual(0, MathHelper.Modulo(-6, 3));
+        }
+
+        /// <summary>
+        /// Product code: MathHelper.Modulo(int, int)
+        /// What: Verifies n mod 1 = 0 for positive n
+        /// Why: Divisor of 1 always yields 0 — guards against divide-by-zero edge case
+        /// </summary>
```
[the remark this answers is not in the document]
Comment (yeelam-gordon) [reply src/modules/peek/Peek.Common.UnitTests/MathHelperTests.cs:85]: Fixed in 3c7f47764a10edb721085ed07ffd89e2107639fb by replacing the misleading divide-by-zero wording with the actual boundary-case rationale. Rebuilt Peek.Common.UnitTests Debug|x64 and reran 37/37 tests passing via vstest.console.exe.
